src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -444,6 +489,8 @@ public InjectState(Long ssrc, MediaStream target, boolean active)
             this.ssrc = ssrc;
             this.active = active;
             this.target = new WeakReference<>(target);
+            this.seqnumOffset = RANDOM.nextInt(Short.MAX_VALUE + 1);
```

Comment:
Short.MAX_VALUE is 2^15, can we use 0xffff (or something with the same value) instead?